sdk/src/main/java/com/podio/sdk/domain/Label.java

```diff
@@ -0,0 +1,22 @@
+package com.podio.sdk.domain;
+
+/**
+ * Created by ianhenry on 6/28/16.
+ */
+public class Label {
+    private final Long label_id = null;
+    private final String text = null;
+    private final String color = null;
```

Comment:
https://developers.podio.com/doc/tasks/get-labels-151534
Currently the api returns the color in hex form so I chose to just represent it as string. I could add an enumeration, but if the hex code ever changed it would break. I think it's best to just return the same thing the api provides.